addOns/automation/src/main/java/org/zaproxy/addon/automation/AutomationJob.java

```diff
@@ -156,12 +155,21 @@ protected void applyParameters(
                                             param.getValue()));
                             continue;
                         } catch (IllegalArgumentException e1) {
-                            progress.error(
-                                    Constant.messages.getString(
-                                            "automation.error.options.badbool",
-                                            this.getType(),
-                                            key,
-                                            param.getValue()));
+                            if (Enum.class.isAssignableFrom(paramType)) {
+                                progress.error(
+                                        Constant.messages.getString(
+                                                "automation.error.options.badenum",
+                                                this.getType(),
+                                                key,
+                                                e1.getMessage()));
```

Comment:
Might be better to use EnumUtils.getEnumList((Class<Enum>) t) here instead of the non i18n'd error message?